README.md

```diff
@@ -91,44 +92,44 @@ This repository offers a wide collection of **ASP.NET Core** Health Check packag
 
 HealthChecks packages include health checks for:
 
-| Package                | Downloads                                                                                                                                                           | Notes                                                                  |
```

Comment:
Please revert format changes of that table. It was already well-formatted to work with.